src/scss/partials/_logo.scss

```diff
@@ -31,39 +31,29 @@ $logo-color--transparent    : $white;
         }
     }
 }
-    .c-logo-img img, .c-logo-img svg {
-        // default logo image height and width (as should be an inline SVG)
-        width: 82px;
-        height: 16px;
 
-        @include media('>=mid') {
-            width: 165px;
-            height: 32px;
-        }
+.c-logo-img img, .c-logo-img svg {
+    // default logo image height and width (as should be an inline SVG)
+    width: 82px;
+    height: 16px;
 
-         // Menulog logo, as it has multiple fill values built in. We just hide the outline on transparent mode.
-        @if ($theme == 'ml') {
-            width: 118px;
-            height: 26px;
-            margin-left: -10.5px; //half of hamburger menu width
+    @include media('>=mid') {
+        width: 165px;
+        height: 32px;
+    }
 
+      // Menulog logo, as it has multiple fill values built in. We just hide the outline on transparent mode.
+    @if ($theme == 'ml') {
+        width: 92px;
+        height: 25px;
+        margin-left: -10.5px; //half of hamburger menu width
 
-            @include media('>=mid') {
-              width: 180px;
-              height: 47px;
-              margin-left: 0;
-              margin-top: 6px;
-            }
-        }
 
-      .c-logo-img {
-        @if ($theme == 'ml') {
-          path:first-child {
-            .c-header--transparent & {
-              display: none;
-            }
-          }
+        @include media('>=mid') {
+          width: 164px;
+          height: 46px;
+          margin-left: 0;
```

Comment:
I know this code was there previously, but just a question as my CSS knowledge is limited.

Why do we specify `margin-left` if we're not adding any margin? Is it purely there to stop it getting inherited from somewhere else?

Reply:
@siggerzz Yep that's right, so if you look above there is a `margin-left: -10.5px;` by default for Menulog, but here for screen widths larger than the mid breakpoint, we basically remove that rule by setting it back to zero